largest_palindrome_product.js

```diff
@@ -5,18 +5,28 @@
  *  @return {Object} an object containing the two factors used to produce
  *                   the palindromeNumber and the palindromeNumber itself.
  */
-module.exports = function(digits){
+module.exports = function (digits) {
   var factor_0 = 0;
   var factor_1 = 0;
   var palindromeNumber = 0;
 
-  // do your work here
+  let maxArr = [0, 10, 100, 1000, 10000, 100000]
```

Comment:
This probably won't be able to scale. Can you think of a way to generate these numbers based on the `digits` parameter passed in?